extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -671,12 +680,16 @@ export class AmpA4A extends AMP.BaseElement {
                   // necessary, because we checked that above.  But
                   // Closure type compiler can't seem to recognize that, so
                   // this guarantees it to the compiler.
-                  if (keyInfo && verifyHashVersion(signature, keyInfo)) {
+                  if (keyInfo &&
+                      this.crypto_.verifyHashVersion(signature, keyInfo)) {
                     user().error(TAG, this.element.getAttribute('type'),
```

Comment:
It should be very infrequent and will result in loss of preferential treatment.  Does dev error send error so we can track frequency?